packages/relay/src/formatters.ts

```diff
@@ -297,6 +299,38 @@ const isHex = (value: string): boolean => {
   return hexRegex.test(value);
 };
 
+/**
+ * Returns the sum of all transfer amounts for the specified account. The amount is negative if the account is charged,
+ * it is positive if the account is receiving it, thus the amount is first negated and then added to the sum.
+ *
+ * @param {TransactionRecord | MirrorNodeTransactionRecord} transactionRecord - The transaction record containing transfer information.
+ * @param {string} accountId - The account ID to filter transfers.
+ * @returns {number} - The sum of transfer amounts for the specified account.
+ */
+const getTransferAmountSumForAccount = (
```

Comment:
why is this added?

Reply:
I think this was added by mistake when I was resolving some conflicts